lib/player.rb

```diff
@@ -0,0 +1,22 @@
+class Player
+  attr_accessor :res_arr
```

Comment:
- **[REQUIRED]** Use attr_readers/attr_writers instead of attr_accessors. The student must be aware that by using it they allow writing that property from outside the class. That variable can be set to anything. <TODO: explain why> ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- res_arr and win_arr were not accessed from outside. I suggest that you either make them local or use attr_reader instead of attr_accessor.